src/Components/NotesPage/FilterBar/FilterBar.jsx

```diff
@@ -0,0 +1,109 @@
+import React, { useState } from 'react';
+import { useFilter } from '../../../context';
+import { priorities } from '../../../data/priorities';
+import { tags } from '../../../data/tags';
+import './FilterBar.css';
+const FilterDropDownbutton = (props) => {
+	return (
+		<div className='dropdown'>
+			<button className='btn btn-filter btn-primary dropdown-btn'>{props.buttonName}</button>
+			<div style={{ transform: `translateX(-${props.scroll}px)` }} className='dropdown-content'>
+				{props.children}
+			</div>
+		</div>
+	);
+};
+
+const FilterBar = () => {
+	const { FilterState, FilterDispatch } = useFilter();
+	const [scroll, setScrolled] = useState('');
+	const scrollHandler = (event) => {
+		setScrolled((scroll) => (scroll = event.target.scrollLeft));
+	};
+
+	return (
+		<aside className='filter-section'>
+			<div className='product-filter-bar'>
+				<div className='filter-items' onScroll={scrollHandler}>
+					<h3>Filter By:</h3>
+					<FilterDropDownbutton
+						key='Sort-By-Date'
+						scroll={scroll}
+						buttonName='Sort By Date'
+						children={
+							<>
+								<label>
+									<input
+										type='radio'
+										name='sort-by-date'
+										checked={FilterState.sortByDate === 'OLDEST_TO_NEWEST'}
+										onChange={() => FilterDispatch({ type: 'SORT_BY_DATE', payload: 'OLDEST_TO_NEWEST' })}
+									/>
+									Sort Oldest to Newest
+								</label>
+
+								<label>
```

Comment:
You can avoid using implicit labels if possible, not all assistive technologies catch it.

Reply:
Okay will have to look into it won't be changing it for this submmision.